public/iframe8.json

```diff
@@ -0,0 +1,131 @@
+{
+    "version": "1.0",
+    "sessions": [
+        {
+            "id": "Diagrid app builder example",
+            "ticket": "b1d3d639c61de96d729902b1ebe1eba4ecd33cf1b89ed04808698d372223595833ebbe6ce9cc76846f3062e8b5f5773a7bac8115fb3e84a342cc71bb9a34ac4815d6311c5fc155f1208eadb32c1322e1c72938ede69486ca5833a3fe82c62a9cd4d8cba24e5e4d-1b8a0ccbbb39e828d7bc2319893bc07a",
+		    "modelViewUrl": "https://sdr7euc1.eu-central-1.shapediver.com"
+        }
+    ],
+    "appBuilderOverride": {
```

Comment:
Using this optional property in the json settings file allows to override the AppBuilder data from the model. This also works even if the model does not have an AppBuilder data output.

Reply:
@MajorMeerkatThe3rd @mathieu-shapediver tagging you here so you know how to use this feature, merging the PR now.